### ISSUE TITLE
Text on input elements - Tailwind UI (Firefox)

### ISSUE THREAD

**Issue by m60digital** [tailwindlabs/tailwindui-issues#439]
Text and password elements containing text appear blank. It looks like their color is set to inherit. This issue can be seen on the Tailwind UI site: https://tailwindui.com/preview

Browser: Firefox 84.0.2 (64-bit)
OS: macOS 11.1 (Big Sur)

**Comment by adamwathan** [comment]
Hmm I can't reproduce this in Firefox here, unless I am not understanding the issue:

![image](https://user-images.githubusercontent.com/4323180/104453723-d5691b00-5572-11eb-880c-c5c8f525459b.png)

Inputs have an explicit `text-gray-900` class set on them in all of our examples typically, is there one specific one maybe where we made a mistake that you're looking at?

**Comment by m60digital** [comment]
Oddly enough, that particular form works for me too. Could you try the Two-column cards with separate submit actions form?

**Comment by adamwathan** [comment]
Seeing black there for me anyways:

![image](https://user-images.githubusercontent.com/4323180/104454067-50cacc80-5573-11eb-839c-7bdbeb448885.png)

But those ones are missing an explicit text color class — maybe it's a dark mode issue? I'll get those updated regardless 👍🏻

**Comment by m60digital** [comment]
I thought it could be an issue with dark mode and tried switching between the two but no joy. This might be related so not sure wether to open a new ticket but the Tailwind UI components page looks like this for me on Firefox:

![image](https://user-images.githubusercontent.com/2155607/104454466-cc417380-559d-11eb-910f-26d7d6ce043e.png)

**Comment by m60digital** [comment]
OK, so I found what might be up here. I checked my Firefox settings and found options for Colours. The settings were as per the attachment. I never really use Firefox and can say for sure that I didn't set these settings to these values myself. They seem like weird defaults, although the option to use these overrides is set to Never.

Switching to Use system colours fixes the issue, but wondering how many others might be affected by this?

![image](https://user-images.githubusercontent.com/2155607/104455083-b2ecf700-559e-11eb-98ac-72bc62506060.png)

**Comment by m60digital** [comment]
I don't think this is an issue with Tailwind, looks more like a Firefox issue. I tried reinstalling Firefox but no joy. I deleted my Firefox preferences and the Text and Background colours in my Preferences have defaulted to back and white respectively, opposite to those settings in the previous attachment. Would love to know how they got switched. 100% confident I didn't do this myself - possibly a prefs file corrupted during an upgrade?

I think if this was affecting a wider audience it would be well documented so hopefully just affecting me. Thanks for looking in to this.
